src/pages/contact-us/_offices.js

```diff
@@ -0,0 +1,251 @@
+
+    @media ${device.tablet} {
+        display: none;
+    }
+`
+const StyledText = styled(Text)`
+    margin-bottom: ${props => props.marginBttom || '0.8rem'};
+    color: ${props => props.color || 'var(--color-black-3)'};
+    text-indent: ${props => props.textIndent || '0'};
+`
+const Splitter = styled.div`
+    background-color: #d8d8d8;
+    height: 0.1rem;
+    width: 98rem;
+`
+export const Offices = () => {
+    useEffect(() => {
+        if (getLocationHash()) {
+            setTimeout(() => {
+                document
+                    .querySelector('#' + getLocationHash())
+                    .scrollIntoView({ behavior: 'smooth', block: 'start' })
+            }, 10)
+        }
+    })
+    return (
+        <Wrapper>
+            <StyledHeader as="h3" align="center" color="--color-black-3" id="our_offices">
+                {localize('Our offices')}
+            </StyledHeader>
+            <OfficesWrapper>
+                <Office>
+                    <ImageWrapper>
+                        <Image
+                            img_name="map-malta.png"
+                            alt={localize('Malta Office')}
+                            width="100%"
+                            height="100%"
+                        />
+                    </ImageWrapper>
+                    <Content>
+                        <StyledHeader as="h4" color="--color-black-3" align="left">
+                            {localize('Malta')}
+                        </StyledHeader>
+                        <StyledText marginBttom="2.1rem">{localize('Office')}</StyledText>
+                        <StyledText width="29.9rem">
+                            <MapIcon></MapIcon>
+                            {localize('Level 3, W Business Centre,')}
+                        </StyledText>
+                        <StyledText marginBttom="1.6rem" textIndent="1.5em">
+                            {localize('Triq Dun Karm Birkirkara BKR9033 Malta.')}
+                        </StyledText>
+                        <StyledText>
+                            <PhoneIcon></PhoneIcon>
+                            {localize('+35621316105')}
+                        </StyledText>
+                    </Content>
+                </Office>
+                <Splitter></Splitter>
+                <Office>
+                    <Content>
+                        <StyledHeader as="h4" color="--color-black-3" align="left">
+                            {localize('Malaysia')}
+                        </StyledHeader>
+                        <StyledText marginBttom="2.1rem">{localize('Cyberjaya office')}</StyledText>
+                        <StyledText>
+                            <MapIcon></MapIcon>
+                            {localize('C-13, iTech Tower, Jalan Impact, Cyber 6,')}
+                        </StyledText>
+                        <StyledText marginBttom="1.6rem" textIndent="1.5em">
+                            {localize('63000Cyberjaya, Selangor, Malaysia.')}
```

Comment:
I think there should be a space after the postal code 
```suggestion
                            {localize('63000 Cyberjaya, Selangor, Malaysia.')}
```

Reply:
It's like the design. I'll check with Alex.

Reply:
I think the design is wrong 😅